### DISCUSSION TITLE
last slash in some links

### DISCUSSION THREAD

**Discussion by sm-khosravi** [vercel/next.js#40203] [Help]
hi
i use next js 12.0.8
in my next.config.js trailingSlash is false because my routes have not slash in the end.
but in my project(my website) i have some links that refers to my weblog (in my weblog i use last slash in the end of routes and trailingSlash is true) but next js removes last slash from links that i used in my project. what should i do?
my weblog domain is the same as  my website domain.

**Comment by icyJoseph** [comment]
Does this happen when using the `next/link` component? If your web-blog is another entity, perhaps using an `<a>` anchor tag directly is a better option? `next/link` and `next/router`'s push, replace and so on are only necessary to keep up the SPA experience, but going to another site is not part of the SPA transitions.
